scripts/launch-prover-service

```diff
@@ -4,20 +4,24 @@
 # client contract will be deployed. This feature is intended for demo or testnet
 # purposes only.
 set -eo pipefail
+NUM_BLOCKS_PER_EPOCH=10
 
 if [[ -z "${ESPRESSO_SEQUENCER_LIGHTCLIENT_ADDRESS}" ]]; then
   if [[ "${ESPRESSO_SEQUENCER_DEPLOY_LIGHTCLIENT_CONTRACT}" == "yes" || "${ESPRESSO_SEQUENCER_DEPLOY_LIGHTCLIENT_CONTRACT}" == "1" ]]; then
     echo "Light client contract address not found, deploying it now."
-    gen-demo-contract --contracts-root-dir="./contracts"
     forge fmt
     forge build
-    forge script DeployLightClientDemoScript --fork-url $ESPRESSO_SEQUENCER_L1_PROVIDER --broadcast
-    ESPRESSO_SEQUENCER_LIGHTCLIENT_ADDRESS=$(cat contracts/broadcast/LightClientDemo.s.sol/*/run-latest.json | jq -r '.receipts[1].contractAddress')
+    forge script contracts/script/LightClient.s.sol:DeployLightClientContractScript  \
+     --sig "runDemo(uint32 numBlocksPerEpoch)" $NUM_BLOCKS_PER_EPOCH \
+            --fork-url $ESPRESSO_SEQUENCER_L1_PROVIDER --broadcast
+    ESPRESSO_SEQUENCER_LIGHTCLIENT_ADDRESS=$(cat contracts/broadcast/LightClient.s.sol/*/runDemo-latest.json | jq -r .receipts[1].contractAddress)
```

Comment:
We may want to get the chain ID from the L1 provider or specify it to make this a bit more robust. It would currently not work correctly if there are deployments for multiple chain IDs.

Reply:
TODO added in f387a99.